doc/00DME0BRD.md

```diff
@@ -0,0 +1,53 @@
+# Business Requirements Document
+
+## Project overview
+
+One of the first elements of a  Modern Operating System (OS) that is perceived by any type of user is the usability of its interaction environment (either command line or graphical user interface based).
+
+Regardless the technologies implied in the general architecture of the OS or the number of applications it can be used for, the we can interact with it is the very first element a user will always perceive.
+
+On top of that the user interface (UI) can also make (or break) the productivity that an OS offers to users of every type.
+
+Finally in modern times general (and non general) users are exposed to a lot of different OS compared to what used to happen back in the early days of home computing pioneers and so they have developed expectations and perceptions towards how such UI interaction should happen.
+
+So far, RISC OS has received little to none improvement and refactoring to follow the new trends on the matter of UI and this can cause two sorts of problems:
+
+a) New users may perceive RISC OS as difficult or not intuitive (according to the new perceptions described above)
+b) RISC OS may not be as customisable and offer an enjoyable experience as much as other OS can do these days
+
+So we thought it would be very useful to introduce a way to allow such changes to be applicable to the RISC OS UI in order to improve and facilitate new users experience on our beloved OS.
+
+At the same time such changes shall not be invasive and shall be kept optional for the userbase that prefers the original way RISC OS has to interact with the user.
+
+RISC OS Desktop and its users would benefit from:
+- A customizable desktop through the addition of a powerful and easy to use Theme Manager
+- More modern and useful functionalities like 
+  - Introducing instant desktop notifications via a new configurable framework
+  - Fast features access via a new global keyboard shortcuts framework
+  - Essential keyboard control of the mouse pointer (for those situations where the mouse may abbandon us!)
+  - and more 
+
+## Success factors
+
+We'd define as success factors for this project:
+- The possibility for a new user to define and use an environment and UI as close as possible to "the modern perceptions created by far more popular OS like MS Windows and Apple macOS". While users acustomed to the old UI shall be let free to keep using the OS as they liked it to be.
```

Comment:
Is this an actual quote? If not, are the "" necessary?

Reply:
good point. The reason for the "" is that the definition is a generic type of phrase I keep getting with conversations with people. Basically in terms of coding saying "it should behave like Apple macOS" is a bit too generic and hard to test. However for now I just captured the thoughts between "", we can review and detail them later (in order to make the requirement testable). For now I just wanted to capture the user feedback.

Reply:
If you think there are better ways to "highlight" non-detailed requirements I am fully open to adopt them :)